.github/workflows/ci.yml

```diff
@@ -8,11 +8,17 @@ on:
   workflow_dispatch:
 jobs:
   build:
-    name: Node.js v${{ matrix.NodeVersion }}
-    runs-on: ubuntu-latest
     strategy:
       matrix:
-        NodeVersion: [14, 16]
+        include:
+          - NodeVersion: 14
+            OS: ubuntu-latest
+          - NodeVersion: 16
+            OS: ubuntu-latest
+          - NodeVersion: 16
+            OS: windows-latest
```

Comment:
If we're already doing Node 16 on Windows, do we need Node 16 on Linux?

Reply:
@D4N14L suggested to build both 14 and 16 on Windows. 

I think we also need to get a Node 18 pipeline enabled, even if we have to skip some Webpack projects.  I will address this in a separate PR

Reply:
You might want to do node 20 already. It becomes the "Active" version fairly soon
![image](https://github.com/microsoft/rushstack/assets/25051063/cd3ae195-d43a-4f49-8327-dde4f5a99d3b)